notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -0,0 +1,1530 @@
+{
```

Comment:
<p>in <code>here</code>, are you linking one of the references below? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I cannot tell which cell you are pointing to as it does not show here.

Reply:
I have changed all of the "here"s to reference indices.